js/apps/filter-page/directives/filters.js

```diff
@@ -10,32 +10,35 @@
 
 				if ($scope.filterFamilies && $scope.filterFamilies.length) {
 					$scope.filterFamilies.forEach((filterFamily) => {
-						const filterFamilyHasTags = filterFamily && Object.hasOwnProperty.call(filterFamily, 'tags') && filterFamily.tags.length;
-						const tags = filterFamilyHasTags ? 
-							filterFamily.tags : 
-							[];
-						let hasMatch = false;
-						
-						$scope.activeFilters.forEach((filter) => {
-							if (!hasMatch) {
-								hasMatch = !!tags
-									.filter((tagName) => findFilterMatch(tagName, filter)).length;
-							}
-						});
-
-						filterFamily.isFilterActive = hasMatch;
+						if (filterFamily) {
+							const filterFamilyHasTags = Object.hasOwnProperty.call(filterFamily, 'tags') && filterFamily.tags.length;
+							const tags = filterFamilyHasTags ?
+								filterFamily.tags :
+								[];
+							let hasMatch = false;
+
+							$scope.activeFilters.forEach((filter) => {
+								if (!hasMatch) {
+									hasMatch = !!tags
+										.filter((tagName) => findFilterMatch(tagName, filter)).length;
+								}
+							});
+
+							// This should probably be refactored to be immutable
```

Comment:
You could start by moving the generation of `hasMatch` to a different function.

Reply:
@ascire I think thats a good idea. I wasn't going for a refactor here, this should be it's own method.